docs/userguide/apps.rst

```diff
@@ -98,6 +99,34 @@ Any Parsl app (a Python function decorated with the ``@python_app`` or ``@bash_a
 3. walltime: (int) This keyword argument places a limit on the app's
    runtime in seconds. If the walltime is exceed, Parsl will raise an `parsl.app.errors.AppTimeout` exception.
 
+Serializing Functions from Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Parsl can create Apps directly from functions defined in Python modules.
+Supply the function as an argument to ``python_app`` rather than creating a new function which is decorated.
+
+.. code-block:: python
+
+    from module import function
+    function_app = python_app(function, executors='all')
+
+``function_app`` will act as Parsl App function of ``function``.
+
+It is also possible to create wrapped versions of functions, such as ones with pinned arguments.
+Parsl just requires first calling :meth:`~functools.update_wrapped` with the wrapped function
```

Comment:
what happens if you don't do this?

Reply:
I should test to be sure. I think we get errors when a function lacks a name, but might be confused with those errors originating from a code which uses Parsl and not Parsl itself.